### ISSUE TITLE
Mob Crusher generating stacks of redstone from mobs

### ISSUE THREAD

**Issue by sharkbite3105** [Buuz135/Industrial-Foregoing#228]
Industrial Foregoing Version: 1.7.0
TeslaCoreLib Version: 1.12-1.0.12
Mob Crusher is generating 1-2 stacks of redstone for each mob it kills, in addition to the normal drops
Edit: had the mobs not spawning with the crusher running and redstone was still appearing in the item pick up area
![2018-02-17_02 08 09](https://user-images.githubusercontent.com/36560028/36338879-ab792180-1387-11e8-85bc-2ebd3c17ceb1.png)

**Comment by Buuz135** [comment]
Do you have something that drops redstone nearby?

**Comment by sharkbite3105** [comment]
no the mob farm is separate from the rest of everything.

**Comment by Seivee** [comment]
We're using cursed earth to spawn mobs, so the witches do drop redstone

**Comment by Buuz135** [comment]
Yeah, witches drop redstone but stack at a time?

**Comment by sharkbite3105** [comment]
no idea, we are in a sky island map and the mob farm is stand alone on its own island, we haven't had this happen since the other night,

**Comment by sharkbite3105** [comment]
no it hasnt happened again, we did have it running after this incident, but there is no way we had dropped the amount of redstone the thing produced, it is connected to a drawer controller via the ender chest, the first time i noticed it there was 34k redstone blocks, i did not turn the mob crusher off and maybe 5-10 minutes later it was almost at 40k redstone blocks,

**Comment by Buuz135** [comment]
I will close the issue now, report if there is more cases

**Comment by phit** [comment]
doing the same thing for us but with clay, killing any mobs gives you stacks of clay..

industrialforegoing-1.12.2-1.8.2-121.jar

**Comment by azziplekkus** [comment]
Can comfirm this is a real issue. Had the same today on a servers, and the admins on the servers couldn't wrap their head around it. 
I didn't alter my mob-farm setup for two days, and was checking my drops in the attached chest. All out of a sudden, there were a few redstone in there. Nothing too odd, because witches drop those (and I did get a witch hat a few moments earlier), but them I kept seeing redstone flowing in. 
At first I thought it was a mob that dropped a full stack of redstone, so I asked in chat if there was any mob that dropped a full stack of redstone. 
But it didn't stop there... it kept generating redstone, and it even seemed as if the rate at which the redstone was generated kept speeding up. 
It generated 2 full double chests of redstone, before I cut the power and reapplied it. It then functioned as normal.
